build/azure-pipelines/cli/cli-win32-sign.yml

```diff
@@ -20,16 +20,16 @@ steps:
     displayName: "Use ESRP client"
 
   - ${{ each target in parameters.VSCODE_CLI_ARTIFACTS }}:
-    - task: DownloadPipelineArtifact@2
-      displayName: Download artifacts
-      inputs:
-        artifact: ${{ target }}
-        path: $(Build.ArtifactStagingDirectory)/pkg/${{ target }}
+      - task: DownloadPipelineArtifact@2
+        displayName: Download artifacts
+        inputs:
+          artifact: ${{ target }}
+          path: $(Build.ArtifactStagingDirectory)/pkg/${{ target }}
 
-    - task: ExtractFiles@1
-      inputs:
-        archiveFilePatterns: $(Build.ArtifactStagingDirectory)/pkg/${{ target }}/*.zip
-        destinationFolder: $(Build.ArtifactStagingDirectory)/sign/${{ target }}
+      - task: ExtractFiles@1
```

Comment:
were these whitespace changes intentional? @joaomoreno

Reply:
Yes, prettier formats the entire file for me.